templater/jobs/presubmit/eks-distro/metrics-server-1-X-presubmits.yaml

```diff
@@ -0,0 +1,17 @@
+commands:
```

Comment:
Looks like the fields are in alpha order, which makes sense, but it might be nice if we ordered them more like we used to where the more important fields are first (name, runifchanged,commands,etc)?